_config.yml

```diff
@@ -16,6 +16,7 @@ download_file_names:
   windows:                   "jamulus_3.7.0_win.exe"
   mac:                       "jamulus_3.7.0_mac.dmg"
   android:                   "jamulus_3.7.0_android.apk"
+download_overview_link:      "https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0"
```

Comment:
Can this stay as `latest` so it won't need changing?

Reply:
Yep, good idea, done.